### ISSUE TITLE
Kodierung Spaltensatz: `<cb>` mit `@type="end"` ist Tag Abuse! 

### ISSUE THREAD

**Issue by cthomasdta** [deutschestextarchiv/dtabf#119]
Da hat Torsten Schaßan leider recht: https://listserv.brown.edu/cgi-bin/wa?A2=TEI-L;b2d25e2e.2403
> […] the usage of `<cb type="end"/>` […] seems to be _tag abuse_ to me as [the definition of `cb`](https://tei-c.org/release/doc/tei-p5-doc/en/html/ref-cb.html) clearly says 'marks _the beginning of a new column_ of a text on a multi-column page'.
> 
> Thus, the presented usage of the element seems to be the exact opposite of what the element is meant to be used for.

Er schlägt auch gleich eine gute alternative Lösung vor:

> Why not replace this with something like
> 
> `<milestone type="end" unit="column"/>` ?

**Comment by susannehaaf** [comment]
Ja, stimmt. Aber das betrifft dann sehr viele Stellen in den DTA-Korpora, fürchte ich.
